### DISCUSSION TITLE
30 day trial period option is gone after entering wrong debit card details

### DISCUSSION THREAD

**Discussion by HyperGrapher** [community/community#121823] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I wanted to subscribe copilot and entered a wrong debit card details which didn't have any balance and now my 30 day trial option is not mentioned in the ui after i entered my other card details with balance in it. 

First time i entered the wrong card details, ui would mention that after 30 days github would charge $10. 

Now i entered the correct card details but I didn't proceed with it, because i would like to have 30 days trial option. 

it would be great if i have 30 days trial period.  Please can someone fix it for me?

**Comment by davecheney** [comment, marked as answer]
I am sorry, we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact
